tests/scripts/analyze_outcomes.py

```diff
@@ -328,6 +328,15 @@ def do_analyze_driver_vs_reference(outcome_file, args):
             }
         }
     },
+    'analyze_driver_vs_reference_ffdh_alg': {
+        'test_function': do_analyze_driver_vs_reference,
+        'args': {
+            'component_ref': 'test_psa_crypto_config_reference_ffdh',
+            'component_driver': 'test_psa_crypto_config_accel_ffdh',
+            'ignored_suites': ['dhm'],
+            'ignored_tests': {'test_suite_pkparse': ['DH group family: RFC 7919']}
```

Comment:
I'm not seeing any such test in `test_suite_pkparse`, what am I missing?

Reply:
Oh ok. This can be removed, because I changed the idea to adapt test dependencies instead of removing this test case and left this by mistake (also invalid version with bad test name):

```
/* BEGIN_CASE depends_on:PSA_KEY_TYPE_DH_PUBLIC_KEY:PSA_KEY_TYPE_DH_KEY_PAIR */
void dh_key_family(int group_arg)
```